pkg/webhook/pod/tikv_deleter.go

```diff
@@ -205,7 +205,7 @@ func (pc *PodAdmissionControl) admitDeleteUpTiKVPodDuringUpgrading(payload *admi
 		}
 	}
 
-	if !isTiKVReadyToUpgrade(payload.pod, store, tc.TiKVEvictLeaderTimeout()) {
+	if !isTiKVReadyToUpgrade(payload.pod, store, stores, tc.TiKVEvictLeaderTimeout()) {
```

Comment:
What about checking the replicas as you're doing in the tikv_upgrader.go and only execute L193-L212 if replicas >= 2?

Reply:
update